### PR TITLE
Chore/tomte linters

### PR DESCRIPTION
**Pull request by dvilelaf**
## Proposed changes

- Linter fixing/skipping due to the new linter settings that came with the tomte library.

- Most changes are migrations from old style string formatting to f-strings, done with the flynt tool. Still, not all the fstring related failures have been solved (protocol generators needs some changes for this to be 100% solved) and therefore the corresponding pylint code has not been removed.

- `use-a-generator` and some other errors have been locally disabled for each line instead of keeping it at pylint config level so future errors are reported.

## Fixes

n/a

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../CONTRIBUTING.md) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [x] Lint and unit tests pass locally with my changes and CI passes too
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that code coverage does not decrease.
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: .pylintrc [hunk 1]
```diff
@@ -2,8 +2,7 @@
 ignore-patterns=__main__.py,.*_pb2.py,tac.sh,tac_local.sh
 
 [MESSAGES CONTROL]
-# Skips after C0209 are temporary
-disable=C0103,C0201,C0301,C0302,W0105,W0707,W1202,W1203,R0801,E1136,E0611,C0209,R1710,R1729,C0206,W0237,R1735,W1505,R1732,W1514,R0402,R1714,R1734,W0640,W0621,E0401,R1733
+disable=C0103,C0201,C0301,C0302,W0105,W0707,W1202,W1203,R0801,E1136,E0611,C0209
```
Comment (dvilelaf) [inline .pylintrc:5]: CS209 must remain for now (it has to do with migrating string formatting to f-strings). 

I've used [flynt](https://pypi.org/project/flynt/) to do 90% automatically and I fixed the rest manually but I found that  the protocol generator will rewrite some of these changes as it uses ".format()" for the messages. The code needs several changes to fix this behaviour, and I've already tried some tests locally but I don't want to expand this PR's scope that should be only about linting. If we want to remove CS209 from here I'll open an issue for that.

Comment (angrybayblade) [inline .pylintrc:5]: Please remove C0209, it's the error code for not using the f-string formatting and since you've addressed this you can remove it

Comment (angrybayblade) [reply .pylintrc:5]: resolved https://github.com/valory-xyz/open-aea/pull/365#discussion_r994409381

**Review by angrybayblade — APPROVED** [review]
LGTM, let's wait for CI

File: .pylintrc (hunk 1 above)
Comment (dvilelaf) [reply .pylintrc:5]: No, I cannot remove it due to: https://github.com/valory-xyz/open-aea/pull/365#discussion_r994409381

File: aea/cli/check_packages.py [hunk 2]
```diff
@@ -226,9 +226,10 @@ def find_all_configuration_files(
     config_files = [
         path
         for path in packages_dir.glob("*/*/*/*.yaml")
-        if any(  # pylint: disable=use-a-generator
+        if any(  # pylint:disable=use-a-generator
```
Comment (DavidMinarsch) [inline aea/cli/check_packages.py:229]: typo

**Pull request closed without merging**